elements/nuxeo-app/nuxeo-expired-session.js

```diff
@@ -54,20 +54,24 @@ Polymer({
         -webkit-transform: translateY(0);
       }
 
-      a,
-      a:active,
-      a:visited,
-      a:focus {
+      button.link {
+        text-decoration: underline;
```

Comment:
Same here, we should remove the underline style to match what we have currently.

```suggestion
   color: var(--nuxeo-link-color, #3a3a54)
```